src/Altinn.AccessManagement.Core/Helpers/RightsHelper.cs

```diff
@@ -104,12 +126,18 @@ public static List<Detail> AnalyzeDelegationAccessReason(Right right)
                 if (delegationPolicySources.Any())
                 {
                     string delegationRecipients = string.Join(", ", delegationPolicySources.SelectMany(delegationPolicySource => delegationPolicySource.PolicySubjects.SelectMany(policySubjects => policySubjects)));
+                    List<AttributeMatch> attributeMatches = new List<AttributeMatch>();
+                    var policySubjects = roleAccessSources.SelectMany(roleAccessSource => roleAccessSource.PolicySubjects);
+                    foreach (List<PolicyAttributeMatch> attributeMatch in policySubjects)
+                    {
+                        attributeMatches.Add(attributeMatch.First());
```

Comment:
Same here, add all attributes

Reply:
Changed from Add to AddRange and turned it into a helper method